Dockerfile

```diff
@@ -1,10 +1,13 @@
 #
 # Define the base image icetray version
 #
-ARG ICETRAY_VERSION=v1.7.2-ubuntu22.04-X64
+ARG ICETRAY_VERSION=v1.9.1-ubuntu22.04-X64
 
 FROM icecube/icetray:icetray-prod-$ICETRAY_VERSION as prod
 
+RUN mkdir -p /opt/i3-data/baseline_gcds && \
+    wget -nv -N -t 5 -P /opt/i3-data/baseline_gcds -r -l 1 -A *.i3* -nd http://prod-exe.icecube.wisc.edu/baseline_gcds/ && \
+    chmod -R u+rwX,go+rX,go-w /opt/i3-data/baseline_gcds
```

Comment:
Is it better to bind-mount these GCDs in, if we have CVMFS available?  Though, are these needed only on the client, or also on the server?

Reply:
> Is it better to bind-mount these GCDs in, if we have CVMFS available? Though, are these needed only on the client, or also on the server?

In #244 I was introducing the stating mechanisms for GCDs, then having them on CVFMS will be sufficient (falling back to http retrieval from `prod-exe`). But at the moment they are needed both on the server and the client. I hope I'll be able to finish that PR by January.

Reply:
Then I guess this is fine for now, since I believe the server uses this image without CVMFS.  (unless @ric-evans tells me I'm wrong)

Reply:
the central server currently does not have access to CVMFS

Reply:
On one hand, I like putting the data in the image so that it is versioned. On the other hand, we may be carrying extra weight. @tianluyuan how much does this change increase the image size?